db/schema.rb

```diff
@@ -91,8 +91,13 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "discarded_at"
+    t.uuid "profile_id"
```

Comment:
where is the `profile_id` field coming from? its not mentioned in the migration above ^ is there another migration to add to this PR ?

Reply:
See subsequent commit. I ran a migration that was abanded/superseded by this :)